packages/EasyUtils/src/CreditCard/CreditCardNumberValidator.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyUtils\CreditCard;
+
+final class CreditCardNumberValidator
```

Comment:
Why did you decide to not use the external package?

Reply:
Because code of this library was too bad and incompatible with PHP 8. 
Considering this and that code size is not very big, I decided to clean it, adopt it and not to require external library.